ignite/contrib/handlers/neptune_logger.py

```diff
@@ -26,6 +27,8 @@
     "global_step_from_engine",
 ]
 
+INTEGRATION_VERSION_KEY = "source_code/integrations/neptune-pytorch-ignite"
```

Comment:
```suggestion
_INTEGRATION_VERSION_KEY = "source_code/integrations/neptune-pytorch-ignite"
```

@twolodzko can you please explain why this is necessary ?

Reply:
Mostly for debugging purposes, we want to track package versions along the metadata. The code works on the client-side, so in some cases, this may be the only way for us to verify what version of the code produced a particular result.

Reply:
OK, in this case please put it inside the class as a private variable. Thanks !

Reply:
Done.